src/ripple/app/consensus/RCLValidations.cpp

```diff
@@ -191,9 +192,26 @@ handleNewValidation(Application& app,
     if (masterKey)
     {
         ValStatus const outcome = validations.add(calcNodeID(*masterKey), val);
+
         if(j.debug())
             dmp(j.debug(), to_string(outcome));
 
+        if (outcome == ValStatus::conflicting && j.fatal())
```

Comment:
Couple thoughts here:
1. Consider this condition to be one of the criteria to be added to nUNL @pwang200  (future, but if we agree we should open an issue to track)
2. Consider this condition to be folded into peer selection/overlay enhancement - this is a byzantine behavior and we should consider disconnecting from this node or at least not connect to this node. Currently, this might not be a good idea but in the future when we are actively relaying validations from untrusted peers, we should consider keep records and potentially not connecting to these bad nodes. My argument here is that this allows us to eliminate bad nodes from the network.